src/IFrameNavigator.ts

```diff
@@ -469,18 +453,15 @@ export default class IFrameNavigator implements Navigator {
         }
     }
 
-    private handleToggleLinksClick(event: MouseEvent): void {
-        event.preventDefault();
+    private handleToggleLinksClick(event: MouseEvent | TouchEvent): void {
         this.hideTOC();
-        if (!this.checkForIFrameLink(event)) {
-           this.toggleDisplay(this.links);
-           this.toggleDisplay(this.linksBottom);
-        }
+        this.toggleDisplay(this.links);
+        this.toggleDisplay(this.linksBottom);
+        event.preventDefault();
```

Comment:
For this `preventDefault` call and the two that follow, should you also be calling `stopPropagation` here? It might not strictly make any difference now but it may avoid subtle issues later.